efficientdet/keras/infer.py

```diff
@@ -60,12 +63,25 @@ def main(_):
   model.load_weights(tf.train.latest_checkpoint(FLAGS.model_dir))
   model.summary()
 
-  @tf.function
-  def f(imgs):
-    return model(imgs, training=False, post_mode='global')
+  class ExportModel(tf.Module):
+    def __init__(self, model):
+      super().__init__()
+      self.model = model
+
+    @tf.function
+    def f(self, imgs):
+      return self.model(imgs, training=False, post_mode='global')
 
   imgs = tf.convert_to_tensor(imgs, dtype=tf.uint8)
-  boxes, scores, classes, valid_len = f(imgs)
+  export_model = ExportModel(model)
```

Comment:
It is okay for this CL, but maybe later we should provide all TF2 savedmodel as well.

Reply:
How about export saved model to TensorFlow hub